### DISCUSSION TITLE
[Proposal] Showcase Island

### DISCUSSION THREAD

**Discussion by iKetsi** [sunflower-land/sunflower-land#2341] [Ideas]
Greetings, would like to suggest the following

A mini island plot mint, apart from main-island , with no nodes in it, just to be used as showcase where we can place NFTs while creating more space in the main land (would also help for when magic beans arrive ) and a chance to burn some sfl/rss maybe as well^

Could also be upgradeable, but its max size shouldnt be to big in my mind, or there could be like 2 or 3 of them in different sides of the map!
Also, since this post comes from discord chat and to me its like midnight, would be nice if this gets any attention to post more ideas on it should anyone like to comment^ 

Have fun everyone^

**Comment by SarielWang93** [comment]
Mungkin NFT yang menampilkan item lain seperti gambar, musik harus didukung di sini. seperti sepatu lari atau kera yang membosankan

**Reply by raoraoxu** [reply]
It's a good idea to show your blue chip nft (BAYC, CryptoPunks) in a picture frame of about 1000

**Comment by VZozul** [comment]
YES! We want small and bigger sand islands. Can be made 3 sizes, the corresponding 10, 20, 30 gold! So that you can put a Palm there, a ball .. Heart of Davy Jones .. different bears .. in general, everything that is decorative.

**Comment by alfa625** [comment]
I don't see why I need another island. I would like to decorate my farm with NFT. I think magic beans are a bad idea in that they reduce the flexibility of decorating. There will be players who will refrain from decorating for the sake of mining.
